main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"encoding/csv"
+	"strconv"
+	"fmt"
+	"net/http"
+	entity "academy-go-q42021/pkg/entity"
+    repo "academy-go-q42021/pkg/repo"
+	server "academy-go-q42021/pkg/server"
+	"os"
+	"log"
+)
+
+type Items []entity.Item
```

Comment:
By convention, every exported stuff (function, variable, constant, etc) should have a description
```go
// Items - Description here....
type Items []entity.Item
```